src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setError] = useState(false);
+  const [isLoaded, setLoaded] = useState(false);
+
+  useEffect(() => {
+    setLoaded(false);
+    setError(false);
```

Comment:
i think you can remove this line, because it's by default false and you didn't change it before

Reply:
Agree